### PR TITLE
Run containers as non-root

### PR DESCRIPTION
**Pull request by agrimmer**
This PR adds securityContext properties to all services of the Keptn control-plane.

Out of scope of this PR:
- helm-service
- jmeter-service
- platform-support/openshift-route-service

Signed-off-by: Andreas Grimmer <andreas.grimmer@dynatrace.com>

### CODE REVIEW

File: api/Dockerfile [hunk 1]
```diff
@@ -58,5 +59,8 @@ ENV GOTRACEBACK=all
 #travis-uncomment COPY entrypoint.sh /
 #travis-uncomment ENTRYPOINT ["/entrypoint.sh"]
 
+RUN adduser -D nonroot -u 65532
```
[the remark this answers is not in the document]
Comment (agrimmer) [reply api/Dockerfile:62]: unprivileged would be also fine

**Comment by agrimmer** [comment]
Note: The nats Helm chart does not allow to set the securityContext. 
I proposed a PR https://github.com/nats-io/k8s/pull/222
If this gets merged, we should update the NATS chart.

**Review by thschue — APPROVED** [review]
lgtm

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**